.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Version to use for the release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Version to use for further development"
+        required: true
+        default: "X.Y.Z-RIE"
+jobs:
+  Release:
+    environment: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git User # may be removed after https://github.com/actions/checkout/issues/13 is resolved
+        run: |
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "${{ github.actor }}"
+      - name: Set up python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Build
+        run: make compile-with-docker-all
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: ${{ github.event.inputs.releaseVersion }}
+          files: |
```

Comment:
Can we add more info to the release ? Maybe not important, but in the past we've prefixed the Release names with "Release ".

Also, we could probably add a parameter to add as the body of the release, in case we want to add some info there. 
Like
```
- Update Go version of the RIE binary to 1.21.9
```
in [the latest release](https://github.com/aws/aws-lambda-runtime-interface-emulator/releases/tag/v1.18).

Reply:
I added something to do this when setting the release version, but you can only enter in one line of notes (aka you can't use bullet points etc). You can always edit the body after the release though, is that ok?